en/_includes/logging/write-logs.md

```diff
@@ -43,7 +40,6 @@
 
       * `--group-name`: Name of the log group to add records to. If this parameter is not specified, records are added to the [default log group](../../logging/concepts/log-group.md) in the current folder.
       * `--message`: Message.
-      * `--timestamp`: Time when the record is sent.
```

Comment:
This description is totally incorrect.
`timestamp` is the time when logging event actually happens and may defers from current time.

I suggest to correct only parameter description.